gpayments/examples/prueba.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import gpayments
+
+from gpayments.error import APIError
+
+gpayments.client_id = ""
+gpayments.client_secret = ""
+
+#gpayments.log = 'debug'
+
+#auth = gpayments.auth()
+
+# store the access_token from auth.data['access_token'] and it expires in auth.data['expires_in']
+# after that just continue to use the access token as this, no need to auth every time and set the client_id and client_secret
+
+gpayments.access_token = '0uC1m74kONYQgE2h9xHIDSmvqjowQp'
+
+#print("Attempting Account retreive...")
+
+#resp = gpayments.Account.retrieve()
+#print('Success: %r' % (resp))
+
+'''simplecharge=gpayments.SimpleCharge.create(
+    amount=10.99,
+    description='Description',
+    entity_description='Entity Description',
+    currency='usd',
+    credit_card_number='4242424242424242',
+    credit_card_security_code_number=123,
+    exp_month=12,
+    exp_year=2020
+)
+print('Success: %r' % (simplecharge)) '''
+
+
+
+try:
+    chargeid = gpayments.Charge.retrieve("1BdPIDCqnAMAMqhvsVq78xpw")
+    print('Charge: %r' % (chargeid))
+except APIError as e:
+    print (e.message)
+
+exit(0)
+
+try:
+    tony = gpayments.Customer.retrieve("S5p6rB3qkyNPCaU6MhK0a5eWiqm6lGUf")
+    print('Tony: %r' % (tony))
+except APIError as e:
+    print (e.message)
+
+#tony.delete()
```

Comment:
We could remove this file because I believe all examples are already in the README.md file, basically prueba.py was there while I was programming the endpoints to test they were working.

What do you think?

Reply:
Yes. Remove them. It's better don not have any commented code.